setup.py

```diff
@@ -126,6 +133,7 @@
         "distribute": distribute_requires,
         "dev": dev_requires,
         "full": full_requires,
+        "online": online_requires,
```

Comment:
add `online_requires` into `full_requires`

Reply:
I agree with weirui, and we shouldn't expose the online requires to users. 
Plz remove it from `extras_require`